### DISCUSSION TITLE
Pytorch Inductor to Triton-shared real model testing (Resnet50)

### DISCUSSION THREAD

**Discussion by pttuan** [microsoft/triton-shared#174] [General]
Hi Everybody,
I am trying to use the ResNet50 model (pre-trained) and following this path: ResNet50 -> PyTorch Inductor -> Triton-shared. Currently, I am experiencing the same issue as described in https://github.com/microsoft/triton-shared/issues/16 when PyTorch attempts to run pointwise kernels. Since the matmul kernel works with Triton-shared, I wonder if anyone knows how to:

- Pass only the matmul kernel to Triton-shared
- Have the rest use the default CPU backend of Inductor without involving Triton.

Also, has anyone used a real model following the same path with Triton-shared?

Thanks,

**Comment by nhat-nguyen** [comment]
Hi @pttuan! Thanks for your interest in the project, and sorry for the delay in responding. Since the project is still in very early stages, supporting arbitrary models, especially code generated from torch inductor is very limited. We're in the process of adding features and bug fixes and will eventually get to fixing #16. Thanks again!
